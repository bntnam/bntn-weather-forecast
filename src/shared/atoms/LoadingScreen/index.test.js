import React from 'react';

import { render } from '@testing-library/react';

import { LoadingScreen } from '.';

const mockData = {
  state: { loading: true },
};

jest.mock('../../../hooks/useAppContext', () => ({
  useAppContext: () => mockData,
}));

describe('LoadingScreen component', () => {
  test('should match the snapshot when loading', () => {
    const { container } = render(<LoadingScreen />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
