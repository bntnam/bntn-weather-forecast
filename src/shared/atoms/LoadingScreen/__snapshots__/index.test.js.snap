// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingScreen component should match the snapshot when loading 1`] = `
<div
  class="sc-bdfBwQ dtsALZ"
>
  <img
    alt="Spinner"
    class="sc-gsTCUz FTLmA"
    src="spinner.gif"
  />
</div>
`;
